tools/bump_version.sh

```diff
@@ -38,11 +38,16 @@ update_image_versions() {
 
 # Print the help message
 print_help() {
-    echo "Usage: $0 bump_type"
+    echo "Usage: $0 <command> [options]"
     echo ""
-    echo "The bump_type argument must be either \"patch\", \"minor\", or \"major\","
-    echo "which instructs the script to increment the patch, minor, and major part"
-    echo "of the semantic version number of Asterinas, respectively."
+    echo "Commands:"
+    echo "  --docker_version_file [major|minor|patch|date]   Update Docker image version in DOCKER_IMAGE_VERSION"
+    echo "  --docker_version_refs                            Update all Docker version references (except VERSION)"
```

Comment:
Change "Docker version" to "Docker image version". These are two different things.

Remove the "(except VERSION)" part. The content of `VERSION` is NOT a reference to the Docker image version.

So the final revised version looks as follows:

```
Update all references to the Docker image version throughout the codebase
```